jsonnet/components/admission-webhook.libsonnet

```diff
@@ -0,0 +1,213 @@
+local tlsVolumeName = 'prometheus-operator-admission-webhook-tls';
+local admissionWebhook = import 'github.com/prometheus-operator/prometheus-operator/jsonnet/prometheus-operator/admission-webhook.libsonnet';
+
+function(params)
+  local aw = admissionWebhook(params);
+
+  aw {
+    deployment+: {
+      metadata+: {
+        labels+: {
+          'app.kubernetes.io/managed-by': 'cluster-monitoring-operator',
+        },
+      },
+      spec+: {
+        template+: {
+          metadata+: {
+            labels+: {
+              'app.kubernetes.io/managed-by': 'cluster-monitoring-operator',
+            },
+          },
+          spec+: {
+            // TODO(simonpasquier): configure client certificate authority to
+            // enforce client authentication.
+            affinity+: {
```

Comment:
nit:
Can we use https://github.com/prometheus-operator/kube-prometheus/blob/main/jsonnet/kube-prometheus/addons/anti-affinity.libsonnet addon to enforce affinity?

Reply:
:+1: